fby_init

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import sys
+from friskby import DeviceConfig
+import friskby_controlpanel.friskby_settings as settings
+from friskby_controlpanel import FriskbyInterface
+
+
+def error_exit():
+    msg = """
+The fby_init shell script should be invoked with the device id of
+your device as commandline argument:
+
+   fby_init myFriskPI_id
+
+Make sure that the device has been 'unlocked' in the control panel of
+the Friskby server before attempting to initialize. When the
+initialization has completed you will get a url which can be used to
+further monitor and control your sensor.  
+"""
+    sys.exit( msg )
+
+
+def locked_device(device_id):
+    msg = """
+Exception 'KeyError' was raised while configuring the device: %s,
+that probably means that your device is locked on the FriskBY
+server. Please go to the server admin panel and uncheck the 'Locked'
+checkbox for device:%s, observe that the device is automatically
+locked after each download.
+""" % (device_id , device_id)
+    sys.exit( msg )
+
+    
+def wrong_id(device_id):
+    msg = """
+Exception 'ValueError' was raised while configuring the device:
+%s, this probably means that your device ID was wrong - the server did
+not recognize device id:%s. Please make sure the device is already
+preconfigured on the server, and that you have typed the id correctly.
+""" % (device_id, device_id)
+    sys.exit( msg )
+
+#-----------------------------------------------------------------
+    
+if len(sys.argv) < 2:
+    error_exit( )
+
+device_id = sys.argv[1]
+
+
+root_url = settings.ROOT_URL
+sensor_path = settings.SENSOR_PATH
+config_url = root_url + "%s/%s/" % (sensor_path, device_id)
```

Comment:
couldn't there be a function in `settings` that:

```python
@classmethod
def config_url(cls, sensor_path, device_id):
    return '%s%s/%s/' % (self.ROOT_URL, sensor_path, device_id)
```

Then you don't have to care about the concept of `root_url` outside whoever knows about the full `config_url`.